Formula/pipenv.rb

```diff
@@ -73,14 +77,14 @@ def install
     (bin/"pipenv").write_env_script(libexec/"bin/pipenv", env)
 
     output = Utils.safe_popen_read(
-      { "PIPENV_SHELL" => "bash" }, libexec/"bin/pipenv", "--completion", { err: :err }
+      { "_PIPENV_COMPLETE" => "zsh_source" }, libexec/"bin/pipenv", { err: :err }
     )
-    (bash_completion/"pipenv").write output
+    (zsh_completion/"_pipenv").write output
 
     output = Utils.safe_popen_read(
-      { "PIPENV_SHELL" => "zsh" }, libexec/"bin/pipenv", "--completion", { err: :err }
+      { "_PIPENV_COMPLETE" => "fish_source" }, libexec/"bin/pipenv", { err: :err }
     )
-    (zsh_completion/"_pipenv").write output
+    (fish_completion/"_pipenv").write output
```

Comment:
`fish` expects a certain filename pattern for its completions (which differs from `zsh`)
```suggestion
    (fish_completion/"pipenv.fish").write output
```

Reply:
thanks 😊